### ISSUE TITLE
Cannot pass environment variables to module

### ISSUE THREAD

**Issue by ianau74** [MaksymBilenko/terraform-provider-aws-synthetics#9]
It looks like the current module doesn't support passing in environment variables as an argument.

I'm guessing because this module was created before November 16th before environment variables was a feature?

It would be super useful to be able to set up a canary and pass environment variables in.  Right now, you have to upload multiple copies of the same script with a 1 line difference.  However, what I really want is to use the same script over and over again, and pass in a single environment variable.

**Right now, we have to do this:**
url='google.com'
...rest of python script

**Ideal python script:**
url=os.environ.get('URL')
...rest of python script

See: https://aws.amazon.com/about-aws/whats-new/2020/11/amazon-cloudwatch-synthetics-supports-environment-variables/

**Comment by lyd-lyd** [comment]
I agree, this would be useful but as of AWS Provider v3.28, this Canary functionality is now part of the AWS Providers and therefore will need to be worked on as functionality there. I very much doubt that there will be any further work on this code and it'll be archived.

**Comment by MaksymBilenko** [comment]
This repo is about to be archived. But if you would like to still use this one and you need some extra functionality you can open PR for it and I'll merge it for you
